exercises/practice/equilateral_triangle/.docs/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Use Z3 to implement a function that, when given two points on the 2D coordinate 
+plane, produces a third point such that the connection of these points
+results in an equilateral triangle.  However, two points should be output
+because there are two possible "third" points to produce an equilateral triangle.
```

Comment:
This sentence is kind of confusing. Maybe change to "The function will have two outputs because there are two possible "third" points to produce an equilateral triangle when given the first two points."